packages/yourloops/test/integration/pages/profile/hcp-profile.spec.tsx

```diff
@@ -0,0 +1,130 @@
+    country: 'FR',
+    units: { bg: Units.mole }
+  }
+  const preferences: Preferences = { displayLanguageCode: 'fr' }
+
+  beforeAll(() => {
+    mockAuth0Hook()
+    mockUserDataFetch({ profile, preferences, settings })
+    mockNotificationAPI()
+    mockDirectShareApi()
+    mockTeamAPI()
+    mockPatientAPI()
+  })
+
+  it('should render profile page for a french hcp and be able to edit his profile', async () => {
+    const expectedProfile = { ...profile, firstName: 'Jean', lastName: 'Talue', fullName: 'Jean Talue', hcpProfession: HcpProfession.nurse }
+    const expectedPreferences = { displayLanguageCode: 'en' as LanguageCodes }
+    const expectedSettings = { ...settings, units: { bg: Units.gram } }
+    const updateProfileMock = jest.spyOn(UserApi, 'updateProfile').mockResolvedValue(expectedProfile)
+    const updatePreferencesMock = jest.spyOn(UserApi, 'updatePreferences').mockResolvedValue(expectedPreferences)
+    const updateSettingsMock = jest.spyOn(UserApi, 'updateSettings').mockResolvedValue(expectedSettings)
+    await act(async () => {
+      renderPage('/preferences')
+    })
+    checkHCPLayout(`${profile.firstName} ${profile.lastName}`)
+    const fields = checkHcpProfilePage(settings.country)
+    const saveButton = screen.getByRole('button', { name: 'Save' })
+
+    expect(fields.firstNameInput).toHaveValue(profile.firstName)
+    expect(fields.lastNameInput).toHaveValue(profile.lastName)
+    expect(fields.unitsSelect).toHaveTextContent(settings.units.bg)
+    expect(fields.languageSelect).toHaveTextContent('Français')
+    expect(fields.hcpProfessionSelect).toHaveTextContent('Diabetologist')
+    expect(saveButton).toBeDisabled()
+
+    fireEvent.mouseDown(within(screen.getByTestId('profile-local-selector')).getByRole('button'))
+    fireEvent.click(screen.getByRole('option', { name: 'English' }))
+
+    fireEvent.mouseDown(within(screen.getByTestId('profile-units-selector')).getByRole('button'))
+    fireEvent.click(screen.getByRole('option', { name: Units.gram }))
+
+    fireEvent.mouseDown(within(screen.getByTestId('hcp-profession-selector')).getByRole('button'))
+    fireEvent.click(screen.getByRole('option', { name: 'Nurse' }))
+
+    userEvent.clear(fields.firstNameInput)
+    userEvent.clear(fields.lastNameInput)
+    await userEvent.type(fields.firstNameInput, 'Jean')
+    await userEvent.type(fields.lastNameInput, 'Talue')
+
+    expect(saveButton).not.toBeDisabled()
+    await act(async () => {
+      userEvent.click(saveButton)
+    })
+
+    expect(saveButton).toBeDisabled()
+    expect(screen.getByRole('alert')).toBeVisible()
+    expect(updatePreferencesMock).toHaveBeenCalledWith(loggedInUserId, expectedPreferences)
+    expect(updateProfileMock).toHaveBeenCalledWith(loggedInUserId, expectedProfile)
+    expect(updateSettingsMock).toHaveBeenCalledWith(loggedInUserId, expectedSettings)
+  })
+
+  it('should render profile page without professional account number when patient is not french', async () => {
```

Comment:
Is this test usefull? I don't see what is tested in this one that is not already tested in the one above :thinking:

Reply:
It's due to commented code in [packages/yourloops/test/integration/assert/profile.ts](https://github.com/mdblp/blip/pull/941/files#diff-1eb3765c70cd9054c9f2e599d90a61c9df20d33a7900b285b85813183e61dd3a)`. Currently the frProId is not setted anymore, but will be reactivated in a futur PR. It will display the CPS number if the hcp is french. Can we keep this commented code for now ?